### ISSUE TITLE
Normalization in Forecasting Task

### ISSUE THREAD

**Issue by Qingrenn** [Y-debug-sys/Diffusion-TS#28]
Dear authors, I noticed that Minmax normalization is fit on all raw data, not only on training data. This operation may cause some data leakage. What is your opinion on this?

https://github.com/Y-debug-sys/Diffusion-TS/blob/21ca525ed6a0ce4322e05d665c792d2b8dc41f99/Utils/Data_utils/real_datasets.py#L129

**Comment by Y-debug-sys** [comment]
Hi, I think your concern is right. But on the one hand, since all the baselines did  the same processing, it will not affect the comparison in the paper. On the other hand, you can of course just normalize the training data. In this case, there are only two suggestions:
1. Do not clip the data to the interval, such as [-1, 1], during sampling;
2. Or/and try the [RevIN](https://github.com/ts-kim/RevIN) trick.
